actions/interviews.ts

```diff
@@ -185,6 +172,29 @@ export async function createInterview(data: CreateInterview) {
       }
     }
 
+    /**
+     * If no participant identifier is provided, we check if anonymous recruitment is enabled.
+     * If it is, we create a new participant and use that identifier.
+     */
```

Comment:
This comment is no longer reflective of the code that follows. Should be updated to explain the logic better.